Makefile

```diff
@@ -33,10 +39,15 @@ build-plugin.%: fmt vet mod-tidy mkdir.devstream ## Build one dtm plugin, like "
 	${GO_BUILD} -o .devstream/${plugin_name}-${PLUGIN_SUFFIX}.so ${PLUGINS_CMD_ROOT}/${plugin_name}
 
 .PHONY: build-plugins
-build-plugins: fmt vet mod-tidy $(addprefix build-plugin.,$(PLUGINS_NAME)) ## Build dtm plugins only, without core, locally.
+build-plugins: fmt vet mod-tidy $(addprefix build-plugin.,$(PLUGINS_NAME)) ## Build dtm plugins only. Use multi-threaded like "make build-plugins -j8" to speed up.
 
-.PHONY: md5
-md5: md5-core md5-plugins
+.PHONY: build
+build: build-core build-plugins ## Build everything. Use multi-threaded like "make build -j8" to speed up.
```

Comment:
Maybe building also need to generate md5.  
According to the logic, dtm needs to verify md5 when loading the plugin